api/v1beta1/condition_types.go

```diff
@@ -73,7 +73,7 @@ type Condition struct {
 	// This should be when the underlying condition changed. If that is not known, then using the time when
 	// the API field changed is acceptable.
 	// +required
-	LastTransitionTime metav1.Time `json:"lastTransitionTime,omitempty"`
+	LastTransitionTime metav1.Time `json:"lastTransitionTime"`
```

Comment:
We could probably drop the `+required` marker? (`no +optional && non pointer && no omitempty` => `required`, isn't it?)

Reply:
ah okay :)